compiler/one-cmds/one-quantize

```diff
@@ -345,6 +361,31 @@ def _verify_arg(parser, args):
             parser.error(
                 'The same number of src_tensor_name and dst_tensor_name should be given.')
 
+    # Check calibration parameters
+    if oneutils.is_valid_attr(args, 'mode'):
+        if getattr(args, 'mode') == 'percentile':
+            # Check dtype
+            try:
+                min_percentile = float(getattr(args, 'min_percentile'))
+            except ValueError:
+                parser.error('min_percentile must be float')
+            try:
+                max_percentile = float(getattr(args, 'max_percentile'))
+            except ValueError:
+                parser.error('max_percentile must be float')
+        elif getattr(args, 'mode') == 'moving_average':
+            # Check dtype
+            try:
+                moving_avg_batch = int(getattr(args, 'moving_avg_batch'))
```

Comment:
`int` may not raise error for float number. You may use `isinstance(..., int)`.

Reply:
The type of argument is string, so `int` raises an exception.

```
>>> int('0.1')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: '0.1'
```

You might mean

```
>>> int(0.1)
0
```

Reply:
The negative test case in this PR tests if the exception is raised.

Reply:
Ah, the argument type is `string`! Thank you for explanation.